src/pages/editor-page/canvas/toolbar/toolbar.tsx

```diff
@@ -50,7 +50,7 @@ export const Toolbar: React.FC<ToolbarProps> = () => {
 
     return (
         <div className="px-1">
-            <Card className="shadow-none p-0 bg-secondary h-[44px]">
+            <Card className="shadow-none p-0 h-[44px]">
```

Comment:
Same like the prev comment, it change also changes the color in light mode. I would use `dark:` prefix instead